features/raadpleeg-verblijfplaats-met-peildatum/gba/test/overlap-datum-aanvang-gba.feature

```diff
@@ -0,0 +1,108 @@
+#language: nl
+
+Functionaliteit: test dat raadplegen historie met periode bij een opeenvolgende verblijfplaatsen met gelijke of overlappende datum aanvang correct wordt geleverd
+
+    Achtergrond:
+      Gegeven adres 'A1' heeft de volgende gegevens
+      | gemeentecode (92.10) | straatnaam (11.10) |
+      | 0800                 | Korte straatnaam   |
+      En adres 'A2' heeft de volgende gegevens
+      | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) | straatnaam (11.10) | naam openbare ruimte (11.15) | woonplaats (11.70) | identificatiecode verblijfplaats (11.80) | identificatiecode nummeraanduiding (11.90) |
+      | 0800                 | 0800010000000002                         | Korte straatnaam   | Officiele straatnaam         | Testdorp           | 0800010000000002                         | 0800200000000002                           |
+
+  Rule: een verblijfplaats wordt niet geleverd wanneer de bekende datum aanvang volgende verblijfplaats gelijk is aan (of eerder) dan de datum aanvang van het verblijf
+
+    Scenario: wel leveren wanneer de datum aanvang van de technisch gewijzigde verblijfplaats later is dan de aanvang oorspronkelijke
+      Gegeven de persoon met burgerservicenummer '000000012' is ingeschreven op adres 'A1' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+      | 0800                              | 20020701                           |
+      En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) | aangifte adreshouding (72.10) |
+      | 0800                              | 20101014                           | T                             |
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde                |
+      | type                | RaadpleegMetPeildatum |
+      | burgerservicenummer | 000000012             |
+      | peildatum           | 2010-01-01            |
+      Dan heeft de response verblijfplaatsen met de volgende gegevens
+      | gemeenteVanInschrijving.code | gemeenteVanInschrijving.omschrijving | datumAanvangAdreshouding | datumAanvangVolgendeAdreshouding | straat           | naamOpenbareRuimte   | woonplaats | adresseerbaarObjectIdentificatie | nummeraanduidingIdentificatie |
+      | 0800                         | Hoogeloon, Hapert en Casteren        | 20020701                 | 20101014                         | Korte straatnaam |                      |            |                                  |                               |
+
+    Abstract Scenario: niet leveren wanneer de datum aanvang van de technisch gewijzigde verblijfplaats is <omschrijving> de aanvang oorspronkelijke
```

Comment:
Hiermee zeggen we dus dat de persoon eerder op het adres is gaan wonen? In deze scenario is het maar drie dagen, maar het zou zomaar ook een jaar eerder kunnen zijn. Is dit correct?